### ISSUE TITLE
External content in PGP-encrypted messages is blocked, TB bug 1931550, caused by TB bug 1925929

### ISSUE THREAD

**Issue by albotroz** [Betterbird/thunderbird-patches#376]
Linux Mint 22 Cinnamon 6.2.9
Betterbird 128.5.0esr-bb18 (64-Bit) (Flatpak)

Since the last update of Betterbird I have the message “external content of this mail has been blocked” in almost every mail - and external content is blocked
I can only close the message and not click Allow like before. External content is still blocked.

The message also appears when I activate external content in the Betterbird settings. External content is still blocked.

In short: external content is blocked in every mail, without the option to allow it as before. And the general setting (allow external content) is ignored.

Edit:
That doesn't work either if I deactivate PGP in Betterbird.

**Comment by Betterbird** [comment]
You didn't mention that this is about remote content in PGP-encrypted messages. There's a long upstream discussion:
https://bugzilla.mozilla.org/show_bug.cgi?id=1931550

We don't intend to action this separately from what TB upstream will offer. Or maybe we will if someone funds it. In that case we can re-open the ticket.

**Comment by Betterbird** [comment]
We're looking into it, we had too many requests for it.

**Comment by Betterbird** [comment]
Try this version:
https://www.betterbird.eu/downloads/WindowsInstaller/betterbird-128.5.0esr-bb18-latest-build7.en-US.win64.installer.exe
Release notes will the updated within 10 minutes.

Folks on Linux, please let us know.

**Comment by Betterbird** [comment]
Note the release notes:
CHANGED (build 7) - Remote content now allowed in encrypted messages after unblocking on a RED banner. **Use at you own risk as decrypted content of the message may be leaked to an attacker** [(EFAIL attack)](https://en.wikipedia.org/wiki/EFAIL).
